apps/backend/fixtures/lambda/parser.json

```diff
@@ -1 +1 @@
-{ "eventType": "all" }
\ No newline at end of file
+{ "Records": [] }
```

Comment:
?

Reply:
This is the shape of the data that we get when the parser lambda is triggered by updates in the S3 bucket. If we receive an empty Records object then we will do as we did before and check the S3 bucket for any discrepancies between the bucket and the db and then parse. (This is so we can still call 'make parse' locally...)